test/cqlpy/test_distinct.py

```diff
@@ -74,7 +74,13 @@ def test_distinct_limit(cql, test_keyspace):
             assert n == len(results)
             assert set(results).issubset(set(all))
 
-# Test combination of SELECT DISTINCT, COUNT, GROUP BY and LIMIT. 
+# Test combination of SELECT DISTINCT and PER PARTITION LIMIT
+def test_distinct_and_per_partition_limit(cql, table1):
+    # reproduces #15109 (DISTINCT + PER PARTITION LIMIT is not allowed)
```

Comment:
Nitpick: I suggest combining this comment line into the comment line in front of the test. It's nice that the comment in front of the test doesn't keep you in suspense and tells you "combination is not allowed", not just "test combination", and also mentions the issue it reproduces. Then you won't need the other comment line inside the test.